installer/mod.ts

```diff
@@ -241,6 +241,11 @@ export async function install(
     }
   }
 
+  // if install local module
+  if (!/^https?:\/\//.test(moduleUrl)) {
+    moduleUrl = path.resolve(moduleUrl);
+  }
```

Comment:
This seems brittle. Can you make this into a standalone function and ideally add a unit test?